airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/config_migrations.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import logging
+from typing import Any, List, Mapping
+
+from airbyte_cdk.config_observation import create_connector_config_control_message
+from airbyte_cdk.entrypoint import AirbyteEntrypoint
+from airbyte_cdk.sources import Source
+from airbyte_cdk.sources.message import InMemoryMessageRepository, MessageRepository
+
+logger = logging.getLogger("airbyte_logger")
+
+
+class MigrateAccessTokenToCredentials:
+    """
+    Facilitates the migration of configuration at runtime.
+    This migration maintains backward compatibility with the previous version by creating a new property.
+    When reverting to the previous source version, the connector will use the old property `access_token`.
+
+    Starting from version `0.3.0`, the `access_token` property is relocated to the `credentials` property.
+    """
+
+    message_repository: MessageRepository = InMemoryMessageRepository()
```

Comment:
Can we remove this since it's not used anymore?

Reply:
Removed